src/cogs/general.py

```diff
@@ -1,16 +1,342 @@
+                f"AFK: {member.voice.afk}```",
+                inline=False,
+            )
+        for activity in member.activities:
+            if isinstance(activity, Spotify):
+                party = await self.determine_party_host(member, activity)
+                artists = ", ".join(activity.artists)
+                userinfo.add_field(
+                    name="Spotify",
+                    value=f"```Song: {activity.title} - {artists} | ({activity.album})\n"
+                    f"Duration: {activity.duration}\n"
+                    f"Party-Host: {party}\n"
+                    f"Track ID: spotify:track:{activity.track_id}```",
+                    inline=False,
+                )
+
+        userinfo.set_footer(text=f"Requested by {ctx.author}", icon_url=ctx.author.avatar.with_size(4096).url)
+        await ctx.send(embed=userinfo)
+
+    @command(name="serverinfo", brief="Shows a bunch of stuff about the guild.")
+    @guild_only()
+    async def serverinfo(self, ctx: Context):
+        days_ago = (datetime.today().replace(tzinfo=ctx.guild.created_at.tzinfo) - ctx.guild.created_at).days
+        mfa = "False" if ctx.guild.mfa_level == 0 else "True"
+        description = ctx.guild.description if ctx.guild.description is not None else "No description set."
+        serverinfo = Embed(title=f"{ctx.guild.name} || ID: {ctx.guild.id}", description=f"{description}", color=self.bot.main_color)
+        serverinfo.set_thumbnail(url=ctx.guild.icon.with_size(4096).url)
+        serverinfo.add_field(
+            name="Info",
+            value=f"```Owner: {ctx.guild.owner}\n"
+            f"Created at: {ctx.guild.created_at.strftime('%H:%M:%S, %d.%m.%Y')} ({days_ago} days ago)\n"
+            f"AFK-Timeout: {int(ctx.guild.afk_timeout) / 60} minutes\n"
+            f"System-Channel: {ctx.guild.system_channel}\n"
+            f"Active invites: {len(await ctx.guild.invites())}\n"
+            f"Active bans: {len(await ctx.guild.bans().flatten())}\n```",
+            inline=False,
+        )
+        serverinfo.add_field(
+            name="Stats",
+            value=f"```Bot Count: {len(ctx.guild.bots)}\n"
+            f"Member Count: {len(ctx.guild.humans)}\n"
+            f"Total member count: {len(ctx.guild.members)}\n"
+            f"Role count: {len(ctx.guild.roles)}\n"
+            f"Channel count: {len(ctx.guild.channels)}\n"
+            f"Bitrate limit: {ctx.guild.bitrate_limit / 1000}kbit/s\n"
+            f"Emoji Limit: {ctx.guild.emoji_limit}```",
+            inline=False,
+        )
+        serverinfo.add_field(
+            name="Miscellaneous", value=f"```Large Server: {ctx.guild.large}\nVerification level: {ctx.guild.verification_level}\n2FA: {mfa}```", inline=False
+        )
+        if ctx.guild.premium_tier > 0:
+            serverinfo.add_field(
+                name="Boosting stats", value=f"```Nitro level: {ctx.guild.premium_tier}\nBoost Count: {ctx.guild.premium_subscription_count}```", inline=False
```

Comment:
```suggestion
                name="Boosting stats", value=f"```Boost level: {ctx.guild.premium_tier}\nBoost Count: {ctx.guild.premium_subscription_count}```", inline=False
```